bin/calculate_res_gene_depth_qc.py

```diff
@@ -0,0 +1,82 @@
+import csv
```

Comment:
Can you add:

```
#!/usr/bin/env python3
```

...to the top of this script?

Reply:
Good catch! [Thank you! ](https://github.com/BCCDC-PHL/tbprofiler-nf/pull/14/commits/067b0f6dc533901d7dc196972d189136e8d62de1)